### ISSUE TITLE
Rate limit results in frequent 429 errors

### ISSUE THREAD

**Issue by IgorKrupenja** [buerokratt/DataMapper#114]
To reproduce:

1. Start Training module with `docker compose up`.
2. Open Intents page and click through ~10 intents.

You will get 429 errors in Data Mapper container. Currently the rate limit is 30 requests per minute per IP — and there are only two IPs making Data Mapper requests: for `ruuter-private` and `ruuter-public`. This will be exceeded even sooner when deployed, consider 10 users using Training — or any other module — at the same time.

Since Data Mapper requests are made from ruuter only, using a rate limit is wrong there. Instead, Data Mapper should not be public and requests should be allowed from ruuter IPs only.

With Kubernetes, @varmoh already implemented this. But we want to implement for Docker compose also and then get rid of the dangerous rate limit. We had a call with @1AhmedYasser and @varmoh and briefly discussed several options:

1. Simply bump the rate limit but this feels wrong and is not a proper solution.
2. Build two different versions of Data Mapper from two different branches, similarly to what we do with ruuter. For docker compose branch, bump limit. For deployed branch, remove altogether. But this has the same problem as 1. And is also cumbersome to maintain?
3. Some docker compose configuration setting to limit connections to ruuter IPs (allow list basically). Probably the best solution but needs research.
4. Limit in the application layer, in NodeJS code, likely using [CORS](https://expressjs.com/en/resources/middleware/cors.html#configuring-cors). This should only be enabled when running locally, so likely will need to pass some env variable through Docker compose.

**Comment by IgorKrupenja** [comment]
Agreed on call with Rainer, Varmo and Ahmed to do option 1 for now. But make this configurable as an env variable which can be passed to via the `docker-compose` file.

**Comment by IgorKrupenja** [comment]
Related PRs:

- https://github.com/buerokratt/DataMapper/pull/124
- https://github.com/buerokratt/Buerokratt-Chatbot/pull/1123
- https://github.com/buerokratt/Training-Module/pull/801
- https://github.com/buerokratt/Analytics-Module/pull/370

To test:

- Should be enough to test that Training/Chat/Analytics modules load without errors.

**Comment by jaX10bt** [comment]
Seems to be working as intended
